### PR TITLE
Break and Continue statements used in C/CPP

### LINKED ISSUE #960: C:Break/Continue in C
<hr>

## Description 📜
I would like to give a detailed description on break and continue in C in documentation form

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

### PR DESCRIPTION
**Pull request by am-dusky**
<hr>

## Description 📜
 I had made a documentation on break and continue statements used in C/CPP. I have also included code snippets  in the md file.
My domain is C/CPP , batch no -1
Fixes #960 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

File: C_CPP/break_continue_in_c_cpp.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Break and Continue in C/CPP
+---
+Break and Countiue are *keywords* used in C/CPP.
+## Break
+* There are situtations where we want to jump out of loop instantly, no matter what is written after that.
```
Comment (deeqakkk) [inline C_CPP/break_continue_in_c_cpp.md:5]: instead of "jump out of loop instantly" use "terminate

**Review by deeqakkk — CHANGES_REQUESTED** [review]
Make changes are commented in the code.

File: C_CPP/break_continue_in_c_cpp.md (hunk 1 above)
Comment (am-dusky) [reply C_CPP/break_continue_in_c_cpp.md]: I have made the changes as mentioned . Kindly check thanku

**Comment by am-dusky** [comment]
@Saviour1001 Please review my work

**Review by gunjan-g — COMMENTED** [review]
@deeqakkk please review if the suggested changes has been made

**Comment by Suvraneel** [comment]
@am-dusky Awesome & clean 
Could you add a line to the readme file... And this could be approved

**Comment by am-dusky** [comment]
@Suvraneel  I have updated the readme . Kindly see.

**Comment by Suvraneel** [comment]
@am-dusky  Awesome !! 
Preach this to your batchmates so they learn too :')

**Pull request merged**